src/pywatemsedem/io/ini.py

```diff
@@ -749,3 +749,54 @@ def add_field(ini, section, key, value):
         msg = f"Adding '{value}' to '{key}' in file '{ini}'"
         logger.info(msg)
         file.writelines(content)
+
+
+def get_item_from_ini(ini, section, option, dtype):
+    """Gets an item from an option in a certain section of an ini-file
+
+    Parameters
+    ----------
+    ini: pathlib.Path
+        File path of the ini-file
+    section: str
+        Name of the desired section
+    option: str
+        Name of the desired option
+    dtype: dtype
+        Type of parameter to be read (str, int, float or bool).
+        If another string is giver to this parameter, a
+        :class:`~pywatemsedem.CNWS.CNWSException` is raised.
+
+    Returns
+    -------
+    str or int or float or bool or None
+        Settings value of ini-file. If the option or section does not exist in
+        the ini-file or the value of the section-option None is returned.
+    """
+
+    Cfg = configparser.ConfigParser()
+    if ini.exists():
+        Cfg.read(ini)
+        try:
+            if dtype == str:
+                a = Cfg.get(section, option)
+            elif dtype == int:
+                a = Cfg.getint(section, option)
+            elif dtype == float:
+                a = Cfg.getfloat(section, option)
+            elif dtype == bool:
+                a = Cfg.getboolean(section, option)
+            else:
+                raise TypeError("not a correct Type")
+        except configparser.NoOptionError:
+            msg = f"Option {option} does not exist in ini-file (section {section})"
+            logger.warning(msg)
+            return None
```

Comment:
shouldn't we throw error, otherwise the code can continue with None? I would then handle the error in the code that is using this function. See also https://github.com/watem-sedem/pywatemsedem/blob/master/src/pywatemsedem/io/ini.py#L637

Reply:
indeed, I changed it to a ValueError